packages/start/vite/plugin.js

```diff
@@ -691,16 +691,18 @@ function islands() {
     transform(code, id, ssr) {
       if (code.includes("unstable_island")) {
         let replaced = code.replaceAll(
-          /const ([A-Za-z_]+) = unstable_island\(\(\) => import\("([^"]+)"\)\)/g,
-          (a, b, c) =>
-            ssr
+          /const ([A-Za-z_]+) = unstable_island\(\(\) => import\((("([^"]+)")|('([^']+)'))\)\)/g,
```

Comment:
`(("([^"]+)")|('([^']+)'))` could also be expressed as `((["'`])(.+?)\1)` at the expense of an additional argument for the replacer.